plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -171,7 +171,10 @@ class Bazel(
         }.getOrNull()
 
     private fun getDependencyGraph(projectDir: File, depDirectives: Map<String, BazelDepDirective>): Set<Scope> {
-        val modGraphProcess = run("mod", "graph", "--output", "json", workingDir = projectDir)
+        val modGraphProcess = run(
+            "mod", "graph", "--output", "json", "--disk_cache", "\"\"", workingDir = projectDir
```

Comment:
I did my tests with `run("mod", "graph", "--output", "json", "--disk_cache=", workingDir = projectDir)` and it worked fine.

Reply:
So you mean to remove the "dummy" `"\"\""` argument in favor of adding a `=` after the option name?

Reply:
I changed it as @nnobelis has put it.